scripts/test-hosting.sh

```diff
@@ -1,21 +1,12 @@
 #!/usr/bin/env bash
 set -e
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CWD="$(pwd)"
 
-if [ "${TRAVIS}" != "true" ]; then
-  TRAVIS_COMMIT="localtesting"
-  TRAVIS_JOB_ID="$(echo $RANDOM)"
-  TRAVIS_REPO_SLUG="firebase/firebase-tools"
-fi
+$DIR/set-default-credentials.sh
```

Comment:
the file needs to be sourced

```suggestion
source $DIR/set-default-credentials.sh
```